### ISSUE TITLE
Packet : MlObj Packet - mlobj

### ISSUE THREAD

**Issue by GodnessCookie** [BlowaXD/ChickenAPI#33]
# PacketName
Mlobj

## Packet Usage
This packet is sent when you Add/Remove Objects like Warehouse/Minigames to your miniland.

## Packet Header
`mlobj`

## Packet Content
`{(deleted)} {ItemInstance.Slot} {MapX} {MapY} {ItemInstance.Item.Width} {ItemInstance.Item.Height} {0} {ItemInstance.DurabilityPoint} {0} {(ItemInstance.Item.IsWarehouse)}`

## Few examples
`mlobj 1 1 31 3 4 2 0 7 0 1` - Add Small Warehouse to miniland
`mlobj 0 1 31 3 4 2 0 7 0 1` - Remove Small Warehouse from miniland
`mlobj 1 0 31 3 4 2 0 10 0 1` - Add Medium Warehouse to miniland
`mlobj 1 3 17 23 5 3 0 6000 0 0` - Add Lv 3 Minigame (Stone) to Map X:17 Y:23

## Class
```C#
        [PacketHeader("mlobj")]
        public class MlobjPacket: PacketBase
        {
            public MlobjPacket()
            {
                Slot = Item.Instance.Slot;
                Width = ItemInstance.Item.Width;
                Height = ItemInstance.Item.Height;
                DurabilityPoint = ItemInstance.DurabilityPoint;
                IsWarehouse = ItemInstance.Item.IsWarehouse;

            }


            [PacketIndex(0)]
            public bool deleted { get; set; }

            [PacketIndex(1)]
            public short Slot { get; set; }

            [PacketIndex(2)]
            public short MapX { get; set; }

            [PacketIndex(3)]
            public short MapY { get; set; }

            [PacketIndex(4)]
            public short Width { get; set; }

            [PacketIndex(5)]
            public short Height { get; set; }

            [PacketIndex(6)]
            public byte Unknown { get; set; } // looks like always 0

            [PacketIndex(7)]
            public short DurabilityPoint { get; set; }

            [PacketIndex(8)]
            public byte Unknown2 { get; set; } // looks like always 0

            [PacketIndex(9)]
            public bool IsWarehouse { get; set; }

        }
```

**Comment by Elendan** [comment]
Nice to see someone contributing, there are just 2 things I'd change : 

the class Name : MlobjPacket would actually be better, as we often have to look at the packets, names have to explicity name them

The constructor : As iteminstances are not there yet

But thanks ! <3

**Comment by BlowaXD** [comment]
Hello,

As Elendan say, you should fix that.

But instead of writing the packet class directly in the issue, I would recommend that you create a pull request.

https://help.github.com/articles/creating-a-pull-request/
Here is the pull request documentation :)

**Comment by BlowaXD** [comment]
solved in #40
